src/components/Graph.js

```diff
@@ -1,33 +1,39 @@
+/*
+		I added Radium to this component as well.  Small amount of styling is used as the actual graph generated
+		is a canvas object.  I changed some of the styling in the graph options object to make it easier
+		to understand/read for the user.  I originally did not optimize the coloring of the graph information,
+		and now its an overall better experience.
+ */
+
 import React, { Component } from 'react';
+import Radium from 'radium';
 import {Line} from 'react-chartjs-2';
 import moment from 'moment';
+import RadiumVars from '../RadiumVariables';
 
 class Graph extends Component {
 	constructor(props) {
 		super(props);
 	}
 
 	render(){		
-		let zuluArray = []
-		let dataArray = []
+		let zuluArray = [];
+		let dataArray = [];
 
 		this.props.cryptoData.forEach(el => {
-			zuluArray.push(moment(el.zuluTime).format("DD-MMM HH:mm"))
+			zuluArray.push(moment(el.zuluTime).format("DD-MMM HH:mm"));
 			dataArray.push(el.rate)
 		});
-		// let banana = moment(zuluArray[0]).format("YYYY-MM-DD HH:mm")
-		// console.log("banana", banana);
 		const data = {
 			labels: zuluArray,
 			datasets: [{
-				label: 'Coin value',
+				label: 'Currency value',
```

Comment:
I changed this to make it a clearer label for the user to understand what was being displayed.  Not all cryptocurrencies would be described as a coin and the previous label could be misleading.